datadog_cluster_agent/README.md

```diff
@@ -0,0 +1,59 @@
+# Datadog Cluster Agent Integration
+
+## Overview
+
+This check monitors the [Datadog Cluster Agent][1].
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+The datadog_cluster_agent check is included in the [Datadog Agent][2] package.
```

Comment:
```suggestion
The `datadog_cluster_agent` check is included in the [Datadog Agent][2] package.
```

Reply:
i feel this should be spelled out as Datadog Cluster Agent, but "The `datadog_cluster_agent` check" works as well.